### PR TITLE
Proposed edits to load-fast-enough-for-pwa

### PR DESCRIPTION
**Pull request by mfriesenhahn**
Changes proposed in this pull request:

- Fixed broken links.
- Edits for concision.
- Changed straight apostrophes to curly.

### CODE REVIEW

File: src/site/content/en/lighthouse-pwa/load-fast-enough-for-pwa/index.md [hunk 1]
```diff
@@ -21,26 +20,26 @@ Checklist](https://developers.google.com/web/progressive-web-apps/checklist#base
 
 ## Recommendations
 
-There are two main metrics regarding how users perceive load time:
+There are two main metrics that affect how users perceive load time:
 
-- The page appears visually complete.
-- The page is interactive. If a page appears visually complete at 1s, but the user can't interact with it until 10s, then the perceived page load time is 10s.
+- Time until page appears visually complete.
+- Time until page is interactive. If a page appears visually complete after 1 second, but the user can’t interact with for 10 seconds, the perceived page load time is 10 seconds.
```
Comment (robdodson) [inline src/site/content/en/lighthouse-pwa/load-fast-enough-for-pwa/index.md]: should it be "can't interact with it for 10 seconds" ?

Comment (mfriesenhahn) [reply src/site/content/en/lighthouse-pwa/load-fast-enough-for-pwa/index.md]: Yes, thanks for the catch.

**Comment by mfriesenhahn** [comment]
> I might suggest sticking with straight quotes as a general rule only because we sometimes include quotes in code and it's very easy for a curly quote to sneak in. When that happens code won't compile but in a lot of text editors it can be really tough to tell them apart.

OK. Typographically, curly quotes are usually strongly preferred for copy, but I understand the concern.

**Comment by robdodson** [comment]
Yeah I agree curly quotes read nicer. I had a personal experience where I tried to help a really frustrated person at one of our sandbox events for like an hour and it turned out there had been a curly quote in the text they copy/pasted into their editor. Ever since I've been wary of them 👀

**Comment by robdodson** [comment]
Hey @mfriesenhahn did you mean to close this? I think if you want to push updates to your branch I can merge them.

**Pull request closed without merging**